pages/api/graphql.ts

```diff
@@ -0,0 +1,54 @@
+import type {NextApiRequest, NextApiResponse} from 'next'
+import got from 'got'
+import * as console from "node:console";
+
+export const config = {
+    api: {
+        // Enable `externalResolver` option in Next.js
+        externalResolver: true,
+        bodyParser: false,
+    },
+}
+
+function getGuacQlServerUrl() {
```

Comment:
Minor nit, but shouldn't this be `get GuacGqlServerUrl` everywhere? It doesn't change the functionality, but it is a little more clear (especially since the environment variable is `GUACGQL_SERVER_URL`)